app/controllers/restaurants_controller.rb

```diff
@@ -1,6 +1,6 @@
 class RestaurantsController < ApplicationController
   def show
-    @cuisine = Cusine.find(params[:cuisine_id])
-    @restaurant = @cuisine.restaurants
+    @restaurant = Restaurant.find(params[:id])
+
   end
 end
```

Comment:
given that this is now in React, you can remove all of the logic from this method